pkg/services/correlations/models.go

```diff
@@ -15,7 +15,7 @@ var (
 
 // CorrelationConfigTarget is the target data query specific to target data source (Correlation.TargetUID)
 // swagger:model
-type CorrelationConfigTarget struct{}
+type CorrelationConfigTarget interface{}
```

Comment:
This is automatically marshaled to a JSON. The difference is that `struct{}` represents a**n empty object** and it's marshaled to empty JSON regardless of the properties in the struct. On the other hand `interface{}` represents **any object** and creates properties in marshaled JSON allowing to have arbitrary structures (and it's what we want because it represents a Data Query object which is different for each data source)